tests/e2e/specs/flinkArtifact.spec.ts

```diff
@@ -31,6 +33,25 @@ test.describe("Flink Artifacts", { tag: [Tag.CCloud, Tag.FlinkArtifacts] }, () =
     "udfs-simple.jar",
   );
 
+  const fixturesDir = path.join(__dirname, "..", "..", "fixtures", "flink-artifacts");
+
+  const fileSizes = [
+    {
+      description: "valid size artifact",
+      setupFile: () => artifactPath,
```

Comment:
just returns a string, decided to go with an inline func for consistency with the next object in the fileSizes array